### DISCUSSION TITLE
Image optimization fails while building even so , I have set unoptimized to true.

### DISCUSSION THREAD

**Discussion by OgLuka** [vercel/next.js#38707] [Help] [answered]
Hi, I have difficulty running the build in my next app.  

Error: Image Optimization using Next.js' default loader is not compatible with `next export`. 

Error is following and has some clear solutions but evenso I am using loader: "akamai" and I have set unoptimized to true the error still persists. 

this is my next.config.js : 

/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true, 
}

 module.exports = {
  experimental: {
    images: {
      loader: 'akamai',
      path: '/',
      unoptimized: true,
    },
  },
}

module.exports = {
  async redirects() {
    return [
      {
        source: '/',
        destination: '/theplatform',
        permanent: false,
      },
    ]
  },
  trailingSlash: true,
}

any help will be appreciated.

**Comment by icyJoseph** [comment, marked as answer]
Hi,

You cannot have multiple `module.exports`, they override each other, and the last one applies.

```js

/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  async redirects() {
    return [
      {
        source: "/",
        destination: "/theplatform",
        permanent: false,
      },
    ];
  },
  trailingSlash: true,
  experimental: {
    images: {
      loader: "akamai",
      path: "/",
      unoptimized: true,
    },
  },
};

module.exports = nextConfig
```

**Reply by OgLuka** [reply]
yes, thanks alot i got the answer from you in another discussion, I forgot I had this one posted.
